internal/app/agent/service.go

```diff
@@ -173,6 +180,26 @@ func (a *agent) Control(uuid, cmdStr string) error {
 	return nil
 }
 
+func (a *agent) ServiceConfig(uuid, cmdStr string) error {
+	cmdArgs := strings.Split(strings.Replace(cmdStr, " ", "", -1), ",")
```

Comment:
With this kind of parsing always give in the comment above how the message looks like, so that we have it documented in the code for developers and reviewers.